tests/robot-tests/tests/visual_testing/tables_and_charts.py

```diff
@@ -5,45 +5,49 @@
 
 
 class DataBlockType(Enum):
-    CONTENT_BLOCK = 1,
-    FAST_TRACK = 2,
-    SECONDARY_STATS = 3,
+    CONTENT_BLOCK = (1,)
+    FAST_TRACK = (2,)
+    SECONDARY_STATS = (3,)
```

Comment:
Seems like in the original code the commas were unnecessary - in fact, it was causing them to be tuples rather than ints, so this is why the code has been changed to this. I think the brackets and the commas can be removed?

Reply:
hmm odd, looks like black was responsible for this. I've disabled formatting for this specific enum for now. Also added a `format:python:check` command to output the diff to avoid issues like this in the future